### DISCUSSION TITLE
Security Onion Web Interface

### DISCUSSION THREAD

**Discussion by tashjianr2** [Security-Onion-Solutions/securityonion#12757] [Unsupported Versions]
Hello!

I am new to using Security Onion and had a question regarding accessing the Web Interface. When I start up Security Onion using Virtual Box, and load it up on another Virtual Machine. I see a blank home page that has access to nowhere shown [Here](https://prnt.sc/YAXHQGlB3M72). Could anyone provide some insight on what may be the issue?

**Comment by Ju3T1N** [comment]
Hello Friend,
I had the same issue (a university assignment required me to utilize some pre-made vm to defend with a firewall and security onion) and I came across this issue. I'm guessing this occurred in Firefox, the Web Portal only works in chromium browsers, otherwise the elements will not load.
Hope this helps.
